docs/REST API/rest-api.md

```diff
@@ -7528,6 +7530,66 @@ Returns information about the specified software version.
 }
 ```
 
+## Vulnerabilities
+
+## Software
+
+- [List vulnerabilities](#list-vulnerabilities)
+- [Get vulnerability](#get-vulnerability)
+
+### List vulnerabilities
+
+Retrieves a list of all CVEs affecting software and/or OS versions.
+
+`GET /api/v1/fleet/vulnerabilities`
+
```

Comment:
Looks like we're missing a `#### Parameters` header here above the table, right?

Also, I think we need a `query` parameter with option to search by `cve`.

Reply:
Oh, also, I think we want a `cisa_known_exploit` boolean parameter to enable the known exploit filter in the UI.